docs/support/project-structure.md

```diff
@@ -168,8 +168,8 @@ are separated out into their own folder, see further down.
   package. The Backstage Storybook is found
   [here](https://backstage.io/storybook)
 
-- [`techdocs-cli/`](https://github.com/backstage/backstage/tree/master/packages/techdocs-cli) -
-  Used for verifying TechDocs locally.
+- [`techdocs-cli/`](https://github.com/backstage/techdocs-cli) - Used for
+  verifying TechDocs locally.
```

Comment:
this line should be removed entirely shouldn't it - it's no longer part of the project stucture, at least not `packages` (which is the headline)

Reply:
Yes @freben, you're right. Will remove. :)